app/scripts/AddTrackPositionMenu.jsx

```diff
@@ -19,14 +19,20 @@ function AddTrackPositionMenu(props) {
       >
         <tbody>
           <tr style={{ height: '30px' }}>
-            <td className={classes['add-track-position-other']} />
+            <td
```

Comment:
Random eslint errors. These are spacers in the layout UI element, so they should have aria-hidden.

Reply:
Actually, these fixes should be in a separate PR (where we bump eslint). I just reset the version.